Sources/DoublePinch/DoublePinch.swift

```diff
@@ -1,10 +1,108 @@
+//
+//  DoublePinch.swift
+//  DoublePinch
+//
+
+import os
 import SwiftUI
+import TappableButton
+
+public struct DoublePinchMode: OptionSet, Sendable {
+    
+    public let rawValue: UInt
+    
+    public init(rawValue: UInt) {
+        self.rawValue = rawValue
+    }
+    
+    public static let accessibilityQuickAction = DoublePinchMode(rawValue: 1 << 0)
+    
+    @available(watchOS 11.0, *)
+    public static let doubleTapGesture = DoublePinchMode(rawValue: 1 << 1)
+    
+    public static var all: DoublePinchMode {
```

Comment:
We'll need to reconsider the "all" option as it breaks the regular double tap gesture on my physical series 10. 

I get the "no double tap gesture available" icon when I use the plain `.doublePinch()` modifier on a button:

![incoming-74E8738B-E659-49C2-9B03-221E635FA0CD](https://github.com/user-attachments/assets/140b3bc9-94cb-4b86-b936-d1e11e371742)

I'm not sure we'll be able to build a single modifier that can handle both "gestures" simultaneously. Instead, I think we should offer a "smart" mode where it chooses to use either the `doubleTapGesture` or `accessibilityQuickAction` based on the device model/OS version combinations.

Reply:
Something like this @erikbasargin ? I'm not sure about the naming but this seems alright 🧐

```swift
/// The double-tap gesture mode supported by the current device and operating system.
///
/// This property determines the most appropriate mode for the double-tap gesture
/// based on the availability of APIs (`watchOS 11.0` and later) and the compatibility
/// of the device model (Apple Watch Series 9 or newer).
///
/// - If the device supports the double-tap gesture (`watchOS 11.0+` and Series 9+), 
///   the `doubleTapGesture` mode is returned.
/// - Otherwise, the `accessibilityQuickAction` mode is used as a fallback.
///
/// Use this property to dynamically adapt functionality to the user's device and OS.
public static var supported: DoublePinchMode {
    if #available(watchOS 11.0, *), DoublePinchMode.isDoubleTapGestureSupported {
        doubleTapGesture
    } else {
        .accessibilityQuickAction
    }
}
```